bin/i18n/codeorg_markdown_crowdin.yml

```diff
@@ -1,18 +1,11 @@
 "project_identifier" : "codeorg-markdown"
-"base_path" : "../../pegasus/sites.v3/code.org"
+"base_path" : "../.."
 
 # API Credentials must be loaded from a separate identity file. See
 # https://support.crowdin.com/configuration-file/#split-project-configuration-and-api-credentials
 "api_key" : ""
 
-"preserve_hierarchy": true
-
 files: [{
-  "source": "/public/international/about.md.partial",
-  # cannot use %file_name% and %file_extension% helpers here, because crowdin
-  # doesn't handle multiple extensions like we'd want
-  "translation": "/i18n/%original_path%/about.%locale%.md.partial",
-}, {
-  "source": "/public/educate/curriculum/csf-transition-guide.md",
-  "translation": "/i18n/%original_path%/%file_name%.%locale%.%file_extension%",
-}]
+  "source": "/i18n/locales/source/markdown/**/*.md",
+  "translation": "/pegasus/sites.v3/code.org/i18n/public/%original_path%/%file_name%.%locale%.md.partial",
```

Comment:
@Hamms the `.partial` is added here

Reply:
oooh, I didn't get that this was applied to both of them

Reply:
Yeah, the downside is that `csf-transition-guide.md` will now be `csf-transition-guide.md.partial`. It should be a noop but obviously still unnecessary.

Reply:
I'm increasingly thinking that we might want to rethink the `.partial` extension and just start thinking of this partial syntax as something we generally support in markdown.

Reply:
Oh I like that